apps/cookbook/src/app/examples/checkbox-radio-sizes-example.scss

```diff
@@ -1,53 +1,60 @@
 @import './libs/designsystem/src/lib/scss/utils';
 
-kirby-checkbox {
+$height-measure-width: 80px;
+$default-size: map-get($checkbox-radio-sizes, 'md');
+
+kirby-checkbox,
+kirby-radio {
   margin-bottom: size('xxs');
   background-color: get-color('semi-light');
   position: relative;
 }
 
 /* height measure */
 kirby-checkbox::before,
-kirby-checkbox::after {
+kirby-checkbox::after,
+kirby-radio::before,
+kirby-radio::after {
   height: 100%;
   border: 1px solid get-color('danger');
   position: absolute;
   right: 0;
 }
 
 /* horizontal measure */
-kirby-checkbox::before {
+kirby-checkbox::before,
+kirby-radio::before {
   content: '';
   border-left: 0;
   border-right: 0;
   width: size('xxs') + 1px;
 }
 
 /* vertical measure */
-kirby-checkbox::after {
-  content: 'md: #{ size(xxxl) }';
+kirby-checkbox::after,
+kirby-radio::after {
+  content: 'md: #{ $default-size }';
   border-left: 0;
   border-top: 0;
   border-bottom: 0;
-  line-height: size('xxxl');
+  line-height: $default-size;
   font-size: font-size('xs');
   color: get-color('danger');
   padding-right: size('xxs');
   margin-right: size('xxs') / 2;
   vertical-align: center;
 }
 
-kirby-checkbox.xs::after {
-  content: 'xs: #{ size(l) }';
-  line-height: size('l');
-}
-
-kirby-checkbox.sm::after {
-  content: 'sm: #{ $fat-finger-size }';
-  line-height: $fat-finger-size;
+@each $size, $height in $checkbox-radio-sizes {
+  kirby-checkbox.#{$size}::after,
+  kirby-radio.#{$size}::after {
+    content: '#{$size}: #{$height}';
+    line-height: $height;
+  }
 }
 
-kirby-checkbox ::ng-deep .wrapper {
+kirby-checkbox ::ng-deep .wrapper,
+kirby-radio ::ng-deep .wrapper {
```

Comment:
Could the slotted mixin be used instead of ng-deep?

Reply:
Not really - we are targeting an internal element of the checkbox/radio...